### PR TITLE
Implemented LogAction on the main plugin and Taser broadcast result

### PR DESCRIPTION
**Pull request by iNilo**
This implements https://github.com/Bara20/TroubleinTerroristTown/issues/49
And implements https://github.com/Bara20/TroubleinTerroristTown/issues/43

### CODE REVIEW

**Comment by iNilo** [comment]
I forked from a non compiling version:

error 017: undefined symbol "Command_DetectiveRules"

So you prob can ignore the fail

File: addons/sourcemod/scripting/ttt/ttt.sp [hunk 1]
```diff
@@ -2379,6 +2379,7 @@ public Action Command_SetRole(int client, int args)
 		TeamInitialize(target);
 		CS_SetClientClanTag(target, " ");
 		CPrintToChat(client, g_iConfig[s_pluginTag], "Player is Now Innocent", client, target);
+		LogAction(client,target, "\"%L\" set the role of \"%L\" to \"%s\"",client,target,"Innocent");
```
Comment (Bara20) [inline addons/sourcemod/scripting/ttt/ttt.sp]: usespaceafteracomma? That looks ....

Comment (iNilo) [inline addons/sourcemod/scripting/ttt/ttt.sp]: I'm not following ? what did I do ?

**Comment by iNilo** [comment]
Also implemented https://github.com/Bara20/TroubleinTerroristTown/issues/43

**Comment by iNilo** [comment]
there you go :+1:

**Comment by Bara20** [comment]
Thanks :+1: , merged.

**Comment by iNilo** [comment]
Ymw

**Pull request merged**